.github/workflows/python-package.yml

```diff
@@ -0,0 +1,37 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Build and Test
+
+on: [push, pull_request]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
```

Comment:
Do we need this? When one of the builds fails, we'll usually focus on this one and fix it, we don't need to run the others. Faster feedback, less energy waste, and we can still activate `fail-fast` when really needed while debugging a failed build.

Reply:
I wondered the same thing, but `fail-fast: false` has grown on me after some thought. Surely it depends on the failure mode, but I can imagine a fair number of scenarios where we end up wasting more energy (both human and machine) by not getting the full picture up front. For example, only seeing one failed matrix result may make it unclear if it's a systematic issue across all versions, or if it is exclusive to one particular build. Another example is if we fix one version only to rerun the workflow and find that another version is failing for a different reason. Given that our tests only take ~20 seconds to complete, I don't think there's too much waste involved with letting them all finish. I could be convinced otherwise, however. :)

Reply:
No strong opinion on this. One more argument in favor of not deactivating fail-fast is that it removes a line of code from our config file ;-).

Reply:
In my opinion, it's better to keep `fail-fast` as false because, it is quite nice to have an overview for the different versions, it's esaier to compare if a build fails, and like hemberger said, the test aren't very long.
But in another way, the default value is true, so indeed, we can remove the line to simplify the config file.